scripts/common/install_yumpackages.sh

```diff
@@ -6,6 +6,11 @@ set -ex
 
 yum install --setopt=tsflags=nodocs -y \
     alsa-lib alsa-lib-devel \
+    alsa-lib-devel \
```

Comment:
This duplicates alsa-lib-devel from the previous line, best to stick to a single item per line to avoid this?

Reply:
You're right I was trying to save some lines some time ago by keeping the `-devel` on the same line as the non-`-devel` package but that's error-prone! will clean.